qiskit_sphinx_theme/static/css/theme.css

```diff
@@ -12138,6 +12138,11 @@ article.pytorch-article .tutorials-callout-container .btn.callout-button a {
 .pytorch-right-menu a:hover span.pre {
   color: #6c6c6d;
 }
+#pytorch-right-menu a.reference.internal{
```

Comment:
It looks like we're using `.` rather than `#` for the entries surrounding this. Any reason to prefer an ID?

Reply:
I chose the more specific just in case since the div looks like 
`<div class="pytorch-right-menu" id="pytorch-right-menu">`
But it does also work using the class instead. I can change that to keep the consistency.